src/Common/ClientAbstract.php

```diff
@@ -0,0 +1,87 @@
+<?php
+/**
+ * SocialConnect project
+ * @author: Patsura Dmitry @ovr <talk@dmtry.me>
+ */
+
+namespace SocialConnect\Common;
+
+use InvalidArgumentException;
+
+/**
+ * Class ClientAbstract
+ * @package SocialConnect\Common
```

Comment:
Nitpick: `@package` tags are quite redundant with the use of namespaces.

Reply:
Nice catch 👍